CoreWebServer/src/au/gov/asd/tac/constellation/webserver/resources/constellation_client.py

```diff
@@ -534,8 +534,10 @@ def get_graph_attributes(self, graph_id=None):
             params = {'graph_id':graph_id}
 
         r = self.call_service('get_graph_values', args=params)
-
-        df = pd.read_json(r.content, orient='split', dtype=False, convert_dates=False)
+        content = r.content
+        if isinstance(content, bytes):
+        content = content.decode('utf8')
```

Comment:
I think this line needs to be indented

Reply:
Yes, this does need to be indented.
Error when testing with Jupyter Notebook:
```
IndentationError: expected an indented block after 'if' statement on line 538
```